enterprise/backend/src/metabase_enterprise/sso/api/sso.clj

```diff
@@ -53,4 +56,17 @@
       (log/error e (trs "Error logging in"))
       (sso-error-page e))))
 
+(api/defendpoint PUT "/saml/settings"
+  "Update SAML settings. You must be a superuser or have settings permissions to call this API."
+  [:as {{:keys [saml-enabled] :as new-settings} :body}]
+  (validation/check-has-application-permission :setting)
+  (try
+   (let [settings (dissoc new-settings :saml-enabled)]
+     (db/transaction
+      (setting/set-many! settings)
```

Comment:
I'm surprised not to find any checks on what settings are allowed. Is it intended that this endpoint can be used to set any settings?

Reply:
That's fair. It doesn't really harm anything to have it open like this since it just ends up being a subset of the behavior of `PUT /api/setting`. But I'll add a check to make the function more explicit